### PR TITLE
Commenting info logs in publisher and store

### PR DESCRIPTION
**Pull request by thushara35**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/apipublisher/scripts/apipublisher.js [hunk 1]
```diff
@@ -376,7 +376,7 @@ var apipublisher = {};
             apiJson.put("destinationStats", api.destinationStats);
             apiJson.put("environments", api.environments);
             success = this.impl.manageAPI(apiJson);
-            log.info("=============================================");
+            // log.info("=============================================");
```
Comment (pulasthi) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/apipublisher/scripts/apipublisher.js]: please remove these logs, these should not be there even in commented state

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/extensions/publisher/assets/api/apis/apimanage.jag [hunk 2]
```diff
@@ -93,10 +93,10 @@ require('/modules/publisher.js').exec(function (ctx) {
 					apiData.cacheTimeout= request.getParameter("cacheTimeout","UTF-8");
 					apiData.destinationStats= request.getParameter("destinationStats","UTF-8");
 					apiData.environments = request.getParameter("environments");
-					log.info("+++++++++++++++++++++++++++++++++++++++++++++++");
+					// log.info("+++++++++++++++++++++++++++++++++++++++++++++++");
 					apiProxy.manageAPI(apiData);
-					log.info("=============================================");
-					log.info(apiData);
+					// log.info("=============================================");
+					// log.info(apiData);
```
Comment (pulasthi) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/extensions/publisher/assets/api/apis/apimanage.jag]: please remove these logs, these should not be there even in commented state

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/apipublisher/scripts/apipublisher.js (hunk 1 above)
Comment (thushara35) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/apipublisher/scripts/apipublisher.js]: PR sent: https://github.com/wso2/carbon-apimgt/pull/577

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/extensions/publisher/assets/api/apis/apimanage.jag (hunk 2 above)
Comment (thushara35) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/extensions/publisher/assets/api/apis/apimanage.jag]: PR sent: https://github.com/wso2/carbon-apimgt/pull/577

**Pull request merged**